app/presenters/track.rb

```diff
@@ -47,17 +47,17 @@ def topic_content(topic_key)
       end
 
       def contributing_instructions(topic_key)
-        content_with_trackler_data("./x/docs/md/track/BETTER.md").gsub('TOPIC', topic_key.to_s.upcase)
+        populate_template_with_trackler_data(File.read("./x/docs/md/track/BETTER.md")).gsub('TOPIC', topic_key.to_s.upcase)
```

Comment:
We're including these contributing instructions intentionally on every track-specific topic page, so it's not really "fallback content." It just happens that the contributing instructions template and the fallback content templates live in the same directory and have the same file format.

It's a good point that we probably don't want to define those twice though. In https://github.com/exercism/exercism.io/pull/3475/commits/483e17f870acca16b3cf0f4a25bf080c3eb72ad2, I named the collection of the contributing instructions template and the fallback content templates "shared templates," because they'll show up on multiple track topic pages.